### DISCUSSION TITLE
predict_model for test data displays the metrics like MAE, MSE, RMSE, R2, RMSLE, MAPE. But We will access it

### DISCUSSION THREAD

**Discussion by NajiAboo** [pycaret/pycaret#3063] [Q&A] [answered]
Predict_model metrics is displaying in the Jupyter notebook, But not able to assign it to a variable

**Comment by ngupta23** [comment, marked as answer]
You will have to use pull() right after the predict_model call to assign it to a dataframe

```python
predictions = predict_model(...)
metrics = pull()
```

**Reply by NajiAboo** [reply]
Thanks. Its worked
